src/WingetCreateCLI/Telemetry/Events/CommandExecutedEvent.cs

```diff
@@ -25,7 +26,7 @@ public class CommandExecutedEvent : EventBase
         /// <summary>
         /// Gets or sets the user provided installer URL.
         /// </summary>
-        public string InstallerUrl { get; set; }
+        public IEnumerable<string> InstallerUrls { get; set; }
 
```

Comment:
Do you know if we can safely change the type of this? I believe I have this event registered with installerUrl as a string. Unless you can confirm that this is fine, I don't want to risk breaking the telemetry event

Reply:
Fair enough, joining to a single string